src/styles/blocks/page.css

```diff
@@ -0,0 +1,22 @@
+.page {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
reset and noemalie css
```suggestion
```

Reply:
Thank tou very much. Detailed review with video - just what a newbie needs. Thanks to the instructions, I corrected it, and most importantly I remembered it)